paths.php

```diff
@@ -54,26 +54,18 @@
 // Load Configuration
 // --------------------------------------------------------
 require (CONFIG.'config.php');
-require (INCLUDES.'current_version.inc.php'); 
-
-/*
-
------ NEED to FIND ANOTHER SOLUTION -----
-Add to config.php when found.
------------------------------------------
-Give your installation a unique ID. If you plan on running multiple instances
-of BCOE&M from the same domain, you'll need to give each installation a 
-unique identifier. This prevents "cross-pollination" of session data display.
-
-For single installations, the default below will be sufficient. Otherwise,
-change the variable to something completely unique for each installation.
-*/
-
-
-$installation_id = "";
-if (empty($installation_id)) $prefix_session = md5("BCOEM012345"); 
-else $prefix_session = md5($installation_id);
+require (INCLUDES.'current_version.inc.php');
 
+/** Using an MD5 of __FILE__ will ensure a different session name for multiple

```

Comment:
Or, simply defined in the line above as a blank variable. This will prevent throwing an error.